lib/sqlalchemy/sql/selectable.py

```diff
@@ -3553,7 +3580,7 @@ def lateral(self, name: Optional[str] = None) -> LateralFromClause:
             :ref:`tutorial_lateral_correlation` -  overview of usage.
 
         """
```

Comment:
**mike bayer** (zzzeek) wrote:

remove

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4325